RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -24,17 +24,25 @@
 
 import RecoTracker.IterativeTracking.iterativeTkConfig as _cfg
 
-iterTrackingEarly = _cfg.createEarlySequence("", "", globals())
+#iterTrackingEarly = _cfg.createEarlySequence("", "", globals())
+#for _eraName, _postfix, _era in _cfg.nonDefaultEras():
+#    _era.toReplaceWith(iterTrackingEarly, _cfg.createEarlySequence(_eraName, _postfix, globals()))
+#
+
+iterTrackingEarlyTask = _cfg.createEarlyTask("", "", globals())
 for _eraName, _postfix, _era in _cfg.nonDefaultEras():
-    _era.toReplaceWith(iterTrackingEarly, _cfg.createEarlySequence(_eraName, _postfix, globals()))
+    _era.toReplaceWith(iterTrackingEarlyTask, _cfg.createEarlyTask(_eraName, _postfix, globals()))
+iterTrackingEarly = cms.Sequence(iterTrackingEarlyTask)
 
-iterTracking = cms.Sequence(InitialStepPreSplitting*
-                            trackerClusterCheck*
-                            iterTrackingEarly*
-                            earlyGeneralTracks*
-                            muonSeededStep*
-                            preDuplicateMergingGeneralTracks*
-                            generalTracksSequence*
-                            ConvStep*
-                            conversionStepTracks
+iterTrackingTask = cms.Task(InitialStepPreSplittingTask,
+                            trackerClusterCheck,
+                            earlyGeneralTracks,
+                            muonSeededStepTask,
+                            preDuplicateMergingGeneralTracks,
+                            generalTracksTask,
+                            ConvStepTask,
+                            conversionStepTracks,
+                            iterTrackingEarlyTask
                             )
+iterTracking = cms.Sequence(iterTrackingTask)
+#iterTracking += iterTrackingEarly
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
Another line commented out to show what I replaced.

Reply:
I don't see this line in the previous version.

Reply:
It might have been removed since I started the edits.